static/skin/viewer.js

```diff
@@ -355,7 +359,7 @@ function onClickEvent(e) {
 
     if (isExternalAppUrl || isExternalUrl(target_href)) {
       const possiblyBlockedLink = blockLink(target_href);
-      if ( e.ctrlKey || e.shiftKey ) {
+      if ( e.ctrlKey || e.shiftKey || linkShouldBeOpenedInANewWindow(target) ) {
```

Comment:
That's very interesting, this is why it is also not working with MacOS command (⌘) key. I did not dared to speak about it in the issue. I should. I think we should add at least `e.metaKey` to this list.

Reply:
On Windows keyboards, `metaKey` is the Windows key (⊞). Clicking a link with that key pressed doesn't open a new tab or window (at least not in Firefox under Ubuntu). Should the `e.metaKey` check be combined with a test that the user agent runs on a Mac?

Reply:
No useful feedback to bring here IMHO. Even if a bit cumbersome, it seems you are on the appropriate path.

Reply:
@benoit74 This will require browser sniffing which isn't considered a good practice (see [comments to this answer on StackOverflow](https://stackoverflow.com/a/20087506)). I will simply include the `metaKey` in the list. I don't think that it is used as a modifier key for clicks under Windows/Linux.

Reply:
I'm fine with this solution as well, I doubt anyone will complain about strange behavior under Windows/Linux when they Meta+click on a link. While I'm sure Mac users might.